pypit/armasters.py

```diff
@@ -187,6 +190,24 @@ def save_masters(slf, det, setup):
                            frametype='slit profile')
 
 
+def save_sensfunc(slf, setup):
+    """ Make YAML friendly and write to disk
+    Separate routine as this process is detector independent
+    
+    Parameters
+    ----------
+    slf
+    setup : str
+    """
+    # Sensitivity Function
+    if 'sensfunc' + settings.argflag['reduce']['masters']['setup'][0] not in settings.argflag['reduce']['masters']['loaded']:
```

Comment:
Well, the detector tag isn't sensible as we take the
brightest star in the image.  But,  I suppose we should
keep the calib ID.  Am adding that back.